spec/lib/efile/nj/nj_flat_eitc_eligibility_spec.rb

```diff
@@ -0,0 +1,333 @@
+require 'rails_helper'
+
+describe Efile::Nj::NjFlatEitcEligibility do
+  describe ".eligible?" do
+    context 'when taxpayer satisfies all eligibility checks' do
+      before do
+        allow(Efile::Nj::NjFlatEitcEligibility).to receive(:possibly_eligible?).and_return true
+      end
+
+      context 'when taxpayer is a qualifying child' do
+        let(:intake) {
+          create(
+            :state_file_nj_intake,
+            :df_data_childless_eitc,
+            :claimed_as_eitc_qualifying_child
+          )
+        }
+
+        it 'returns false' do
+          result = Efile::Nj::NjFlatEitcEligibility.eligible?(intake)
+          expect(result).to eq(false)
+        end
+      end
+
+      context 'when nfj and spouse is a qualifying child' do
+        let(:intake) {
+          create(
+            :state_file_nj_intake,
+            :df_data_childless_eitc,
+            :mfj_spouse_over_65,
```

Comment:
[pebble] is it specifically necessary to include the spouse over 65? Can we either mention that in the name of the test somehow so that we know it's an important part of setup? Or can we just use the regular married-filing-jointly trait if it's not necessary, to keep the test setup strictly related to what's under test

Reply:
same thing below as well

Reply:
good point, will make it clear why it's needed

Reply:
It's an interesting question because technically the `:mfj_spouse_over_65` is more realistic. but that part of eligibility is part of what's mocked in that test, so it really has no effect either way and so I went with the simpler `:married_filing_jointly`.